Dockerfile

```diff
@@ -14,8 +14,9 @@ RUN pip3 install sphinx && \
     pip3 install gunicorn && \
     # These lines combat a vulnerability in pyxdg in both python2 and python3
     # https://github.com/advisories/GHSA-r6v3-hpxj-r8rv
-    pip3 install pyxdg==0.26 && \
-    pip install pyxdg==0.26
+    apt -y remove python-xdg \
+      python3-xdg
+
```

Comment:
I honestly do not know what is happening here but seems like it solved the issue.

Reply:
The python package `pyxdg` has been found to contain security vulnerabilities in versions < `0.25`.  I tried manually updating the package to `0.26` (which had solved the vulnerability) but the `anchore` scan still reported the issue.  I am now removing the OS package that `pyxdg` uses to avoid the issue altogether.  This is **not** my preferred solution but ever other thing I tried did not fix it.